src/components/Dashboard/Form/Applicants/Applicants.js

```diff
@@ -40,67 +42,39 @@ const Applicants = ({ reasonsList, disabilityList }) => {
     );
   };
 
+  const issueOnChange = event => {
+    setChosenParent(event.target.value);
+    setFirstReason(null);
+    setSecondReason(null);
```

Comment:
Yeah... There is intentionally here because we use this to reset select's value after changing the value in other select. Probably it requires refactoring but here I've just moved this to the separate method.